backend/api/models/User.py

```diff
@@ -162,7 +166,11 @@ def is_government_user(self):
         """
         Does this user have a government role?
         """
-        if self.roles.filter(Q(is_government_role=True)):
+        is_gov_user = cache.get(f'role-user-{self.id}-gov-role-true')
```

Comment:
again i don't think we want to cache this value in case roles change. Even though a gov user should only ever be the user with gov roles, i still think we should avoid caching this value.